layouts/index.html

```diff
@@ -87,7 +87,8 @@ <h2>{{i18n "featured-post" }}</h2>
         </div>
       </div>
       <div class="row blog-posts">
-        {{ $posts := where .Pages "Section" "blog" }}
+        {{ $p := where .Site.Pages "Section" "==" "blog" }}
+        {{ $posts := where $p ".Params.processed" ">" 0 }}
```

Comment:
The processed tag is a legacy tag from a previous pipeline. It's not present in strapi (and any blog posts being managed there, which are from the last few months and any future ones). So this probably won't work :(

Reply:
Nice catch, thanks. I'll see if I can find another key which can be used for this purpose.

Does that mean I could make a PR to remove all the `processed` entries? Are there other keys which can be removed?

Reply:
Yep, the processed keys are not being used. I would have to take a more in depth look to remember any others, but I can probably get back to you on that later today.

Reply:
Ok, switched to `.Params.author != nil` as the folder entry doesn't have an author but as far as I can tell all blog posts do.